.github/workflows/ci.yaml

```diff
@@ -50,6 +50,6 @@ jobs:
           cache-from: type=registry,ref=${{ env.CACHE_IMAGE_FULL }}
           cache-to: type=registry,ref=${{ env.CACHE_IMAGE_FULL }},mode=max
           context: .
-          file: ./apache.Dockerfile
+          file: ./dockerfiles/apache.Dockerfile
```

Comment:
does this PR move the file, or create a new one? Did you forget to delete the apache.Dockerfile in the root folder?